modmesh/gui/naca.py

```diff
@@ -29,125 +29,24 @@
 Show NACA airfoil shape
 """
 
-import numpy as np
```

Comment:
Move the code from `modmesh/gui/naca.py` to `modmesh/pilot/airfoil/__init__.py`.